sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario6/DirectPDSAPICheckmarxWrapperScenario6IntTest.java

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.integrationtest.scenario6;
+
+import static com.mercedesbenz.sechub.integrationtest.api.TestAPI.*;
+import static com.mercedesbenz.sechub.test.TestConstants.*;
+
+import java.util.UUID;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+
+import com.mercedesbenz.sechub.commons.pds.data.PDSJobStatusState;
+import com.mercedesbenz.sechub.integrationtest.api.IntegrationTestSetup;
+import com.mercedesbenz.sechub.integrationtest.api.TestAPI;
+import com.mercedesbenz.sechub.test.TestFileReader;
+
+/**
+ * Integration test directly using REST API of integration test PDS (means
+ * without sechub).
+ *
+ * @author Albert Tregnaghi
+ *
+ */
+public class DirectPDSAPICheckmarxWrapperScenario6IntTest {
+
+    @Rule
+    public IntegrationTestSetup setup = IntegrationTestSetup.forScenario(Scenario6.class);
+
+    @Rule
+    public Timeout timeOut = Timeout.seconds(600);
+
+    @Test
+    public void pds_techuser_can_start_checkmarx_scan_with_job_configuration_and_PDS_job_is_done() {
+        /* @formatter:off */
+        /* prepare */
+        String json =TestFileReader.loadTextFile("src/test/resources/pds/checkmarx-wrapper/checkmarx-pds-job1.json");
+
+        String createResult = asPDSUser(PDS_ADMIN).
```

Comment:
The name of the test is about the `pds_techuser` executing the job. Here the `PDS_ADMIN` is used. Using the PDS techuser account would be better, because that is what SecHub usually uses to communicate to the PDS.